pkg/scalers/nats_jetstream_scaler.go

```diff
@@ -36,14 +35,17 @@ type natsJetStreamScaler struct {
 }
 
 type natsJetStreamMetadata struct {
-	account                string
-	stream                 string
-	consumer               string
-	consumerLeader         string
+	Account                string `keda:"name=account, order=authParams;triggerMetadata"`
```

Comment:
Is this still required? I mean, when `account` and `accountID` are different, is `accountID` the used one. Maybe they can be optional both individually, but requiring one of them

Reply:
I'm thinking about how to use this new feature, and most probably the name doesn't matter if I specify the accountID (and I think that Account isn't used anymore in the code except for the validation, so it looks as optional)